driver/src/main/java/org/neo4j/driver/internal/handlers/ResetResponseHandler.java

```diff
@@ -54,13 +54,13 @@ public final void onRecord( Value[] fields )
         throw new UnsupportedOperationException();
     }
 
-    private void resetCompleted()
+    private void resetCompleted( boolean success )
     {
         messageDispatcher.unMuteAckFailure();
-        resetCompleted( completionFuture );
+        resetCompleted( completionFuture, success );
     }
 
-    protected void resetCompleted( CompletableFuture<Void> completionFuture )
+    protected void resetCompleted( CompletableFuture<Void> completionFuture, boolean success )
```

Comment:
Do we loose information without using the `RESET` outcome, i.e. `success` parameter?

Reply:
This response handler is used only for `Session#reset()` which is a best-effort call to clear the connection state on the database side. It might succeed, fail or do nothing regardless of a network error. That is why I thought it is okay to simply notify about the completion of the remote call here. `Session#reset()` is also deprecated :)

Does this make sense?